<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UBUVoiceAssistant</string>
  </property>
  <widget class="QPushButton" name="btnConfig">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMute">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>mute/unmute</string>
   </property>
  </widget>
  <widget class="QWebEngineView" name="web" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>761</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbxInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>581</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>type here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeyMycroft">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Sans</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>#Hey Mycroft</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblQuestions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>#questions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
